workflow-examples/src/test/java/com/redhat/parodos/examples/integration/SimpleWorkFlow.java

```diff
@@ -0,0 +1,175 @@
+			assertEquals("simpleSequentialWorkFlow" + WorkFlowConstants.INFRASTRUCTURE_WORKFLOW,
+					simpleSequentialWorkFlowDefinition.getName());
+			assertEquals(WorkFlowProcessingType.SEQUENTIAL.toString(),
+					simpleSequentialWorkFlowDefinition.getProcessingType());
+			assertEquals(WorkFlowType.INFRASTRUCTURE.toString(), simpleSequentialWorkFlowDefinition.getType());
+
+			assertNotNull(simpleSequentialWorkFlowDefinition.getWorks());
+			assertTrue(simpleSequentialWorkFlowDefinition.getWorks().size() == 2);
+			assertEquals("restCallTask", simpleSequentialWorkFlowDefinition.getWorks().get(0).getName());
+			assertEquals(WorkType.TASK.toString(), simpleSequentialWorkFlowDefinition.getWorks().get(0).getWorkType());
+			assertNull(simpleSequentialWorkFlowDefinition.getWorks().get(0).getWorks());
+			assertNull(simpleSequentialWorkFlowDefinition.getWorks().get(0).getProcessingType());
+			assertNotNull(simpleSequentialWorkFlowDefinition.getWorks().get(0).getParameters());
+
+			assertEquals("loggingTask", simpleSequentialWorkFlowDefinition.getWorks().get(1).getName());
+			assertEquals(WorkType.TASK.toString(), simpleSequentialWorkFlowDefinition.getWorks().get(1).getWorkType());
+			assertNull(simpleSequentialWorkFlowDefinition.getWorks().get(1).getWorks());
+			assertNull(simpleSequentialWorkFlowDefinition.getWorks().get(1).getProcessingType());
+			assertNotNull(simpleSequentialWorkFlowDefinition.getWorks().get(1).getParameters());
+
+			// Define WorkRequests
+			WorkRequestDTO work1 = new WorkRequestDTO();
+			work1.setWorkName("restCallTask");
+			work1.setArguments(Arrays.asList(new ArgumentRequestDTO().key("url").value("https://httpbin.org/post"),
+					new ArgumentRequestDTO().key("payload").value("'Hello!'")));
+
+			WorkRequestDTO work2 = new WorkRequestDTO();
+			work2.setWorkName("loggingTask");
+			work2.setArguments(Arrays.asList(new ArgumentRequestDTO().key("user-id").value("test-user-id"),
+					new ArgumentRequestDTO().key("api-server").value("test-api-server")));
+
+			// Define WorkFlowRequest
+			WorkFlowRequestDTO workFlowRequestDTO = new WorkFlowRequestDTO();
+			workFlowRequestDTO.setProjectId(testProject.getId());
+			workFlowRequestDTO.setWorkFlowName("simpleSequentialWorkFlow_INFRASTRUCTURE_WORKFLOW");
+			workFlowRequestDTO.setWorks(Arrays.asList(work1, work2));
+
+			WorkflowApi workflowApi = new WorkflowApi();
+			log.info("******** Running The Simple Sequence Flow ********");
+			WorkFlowResponseDTO workFlowResponseDTO = workflowApi.execute(workFlowRequestDTO);
+
+			assertNotNull(workFlowResponseDTO.getWorkFlowExecutionId());
+			assertNull(workFlowResponseDTO.getWorkFlowOptions());
+			assertNotNull(workFlowResponseDTO.getWorkStatus());
+			assertEquals(WorkStatusEnum.COMPLETED, workFlowResponseDTO.getWorkStatus());
+
+			log.info("workflow finished successfully with response: {}", workFlowResponseDTO);
+			log.info("******** Simple Sequence Flow Completed ********");
+
+		}
+		catch (ApiException e) {
```

Comment:
Why not to use `ApiException` to fail the test? Is there a value in catching it?

Reply:
No value, you're right! I'll remove it